docs/spot-instances.md

```diff
@@ -73,6 +73,7 @@ Launch Template support is a recent addition to both AWS and this module. It mig
     }
   ]
 
+/* The variable worker_groups_launch_template_mixed is not suported from v6.0.0 onwards */
```

Comment:
I think just remove it.

We should make the examples reflect the current version only. If we have to write notes about backwards and forwards compatibility this is too much work and too complicated 😃

Reply:
Sounds good, thanks for the review!